qa/python_models/bls/model.py

```diff
@@ -483,6 +486,13 @@ def test_bls_tensor_lifecycle(self):
                 f"input ({input0}) and output ({output0_pytorch}) didn't match for identity model.",
             )
 
+            # We are seeing intermittent failures in the GPU memory lifecycle
+            # test where the free memory is not the same as the recorded memory.
+            # It is suspected that this is due to the Python garbage collector
+            # not releasing the memory immediately. Calling the garbage
+            # collector here as experiment to see if it fixes the issue.
```

Comment:
If it fixes the issue can we remove the last line?

Reply:
Sure! Removed the last line.